events/wc_race_events.txt

```diff
@@ -655,6 +655,10 @@ character_event = {
 			add_trait = creature_ancient_wind
```

Comment:
`WCCOU.200` is broken.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108637421-ff7ff800-74a3-11eb-97a5-ee29e72a6bf8.png)

</details>